src/testsuite/arith128_test_bcd.c

```diff
@@ -4271,61 +4399,2603 @@ test_bcd_caddec256 (void)
+  j =  (vui128_t) vec_bcdtrunc ((vBCD_t) i, s);
+
+#ifdef __DEBUG_PRINT__
+  print_vint128x ("BCD qword ", i);
+  print_vint128x ("BCD trunc ", (vui128_t) s);
+  print_vint128x ("truncated ", j);
+#endif
+  rc += check_vuint128x ("vec_bcdtrunc:", (vui128_t) j, (vui128_t) e);
+
+  s = (vui16_t) CONST_VINT128_DW (32, 0);
+  e = CONST_VINT128_DW128 ( 0x1011121314151617UL,
+	                    0x202122232425262cUL );
+  j =  (vui128_t) vec_bcdtrunc ((vBCD_t) i, s);
+
+#ifdef __DEBUG_PRINT__
+  print_vint128x ("BCD qword ", i);
+  print_vint128x ("BCD trunc ", (vui128_t) s);
+  print_vint128x ("truncated ", j);
+#endif
+  rc += check_vuint128x ("vec_bcdtrunc:", (vui128_t) j, (vui128_t) e);
+
+  return rc;
+ }
+
+//#define __DEBUG_PRINT__ 1
+int
+test_bcdutrunc (void)
+{
+  vui128_t i, j, e;
+  vui16_t s;
+  int rc = 0;
+
+  printf ("\n%s Vector Decimal Unsigned Truncate\n", __FUNCTION__);
+
+  i = CONST_VINT128_DW128 ( 0x1011121314151617UL,
+	                    0x2021222324252627UL );
+  s = (vui16_t) CONST_VINT128_DW (0, 0);
+  e = CONST_VINT128_DW128 ( 0x0000000000000000UL,
+	                    0x0000000000000000UL );
+  j =  (vui128_t) vec_bcdutrunc ((vBCD_t) i, s);
+
+#ifdef __DEBUG_PRINT__
+  print_vint128x ("BCD qword ", i);
+  print_vint128x ("BCD trunc ", (vui128_t) s);
+  print_vint128x ("truncated ", j);
+#endif
+  rc += check_vuint128x ("vec_bcdutrunc:", (vui128_t) j, (vui128_t) e);
+
+  i = CONST_VINT128_DW128 ( 0x1011121314151617UL,
+	                    0x2021222324252627UL );
+  s = (vui16_t) CONST_VINT128_DW (1, 0);
+  e = CONST_VINT128_DW128 ( 0x0000000000000000UL,
+	                    0x0000000000000007UL );
+  j =  (vui128_t) vec_bcdutrunc ((vBCD_t) i, s);
+
+#ifdef __DEBUG_PRINT__
+  print_vint128x ("BCD qword ", i);
+  print_vint128x ("BCD trunc ", (vui128_t) s);
+  print_vint128x ("truncated ", j);
+#endif
+  rc += check_vuint128x ("vec_bcdutrunc:", (vui128_t) j, (vui128_t) e);
+
+  s = (vui16_t) CONST_VINT128_DW (2, 0);
+  e = CONST_VINT128_DW128 ( 0x0000000000000000UL,
+	                    0x0000000000000027UL );
+  j =  (vui128_t) vec_bcdutrunc ((vBCD_t) i, s);
+
+#ifdef __DEBUG_PRINT__
+  print_vint128x ("BCD qword ", i);
+  print_vint128x ("BCD trunc ", (vui128_t) s);
+  print_vint128x ("truncated ", j);
+#endif
+  rc += check_vuint128x ("vec_bcdutrunc:", (vui128_t) j, (vui128_t) e);
+
+  s = (vui16_t) CONST_VINT128_DW (15, 0);
+  e = CONST_VINT128_DW128 ( 0x0000000000000000UL,
+			    0x0021222324252627UL );
+  j =  (vui128_t) vec_bcdutrunc ((vBCD_t) i, s);
+
+#ifdef __DEBUG_PRINT__
+  print_vint128x ("BCD qword ", i);
+  print_vint128x ("BCD trunc ", (vui128_t) s);
+  print_vint128x ("truncated ", j);
+#endif
+  rc += check_vuint128x ("vec_bcdutrunc:", (vui128_t) j, (vui128_t) e);
+
+  s = (vui16_t) CONST_VINT128_DW (31, 0);
+  e = CONST_VINT128_DW128 ( 0x0011121314151617UL,
+	                    0x2021222324252627UL );
+  j =  (vui128_t) vec_bcdutrunc ((vBCD_t) i, s);
+
+#ifdef __DEBUG_PRINT__
+
```

Comment:
Because this is the bcdutrunc unit test and I copied the debug prints from bcdtrunc?
This is just text to help tell on 31-digit number from the others when debugging.
But I'll change it to BCDutrunc.